pkgs/applications/graphics/freecad/realthunder.nix

```diff
@@ -0,0 +1,8 @@
+import ./generic.nix rec {
```

Comment:
Instead of a loose name like  "generic", I suggest 
```suggestion
import ./make-freecad.nix rec {
```

On the other hand, I truly prefer the `override*` framework.